webapp/src/components/Modals/SellModal/SellModal.tsx

```diff
@@ -0,0 +1,359 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import React, { useState } from 'react'
+import { Link } from 'react-router-dom'
+import { Button, Field, Mana, Message, ModalNavigation } from 'decentraland-ui'
+import { Network, NFTCategory } from '@dcl/schemas'
+import { ethers } from 'ethers'
+import addDays from 'date-fns/addDays'
+import formatDate from 'date-fns/format'
+import isValid from 'date-fns/isValid'
+import { hasAuthorization } from 'decentraland-dapps/dist/modules/authorization/utils'
+import { ContractName } from 'decentraland-transactions'
+import { T, t } from 'decentraland-dapps/dist/modules/translation/utils'
+import {
+  AuthorizationType,
+  Authorization as Authorizations
+} from 'decentraland-dapps/dist/modules/authorization/types'
+import { ChainButton, Modal } from 'decentraland-dapps/dist/containers'
+import { toFixedMANAValue } from 'decentraland-dapps/dist/lib/mana'
+
+import { getContractNames, VendorFactory } from '../../../modules/vendor'
+import { getAssetName, isOwnedBy } from '../../../modules/asset/utils'
+import {
+  getDefaultExpirationDate,
+  INPUT_FORMAT
+} from '../../../modules/order/utils'
+import { ManaField } from '../../ManaField'
+import { parseMANANumber } from '../../../lib/mana'
+import { showPriceBelowMarketValueWarning } from '../../SellPage/SellModal/utils'
+import { Authorization } from '../../SettingsPage/Authorization'
+import { locations } from '../../../modules/routing/locations'
+import { isAuthorized } from '../../SettingsPage/Authorization/utils'
+
+import { Props } from './SellModal.types'
+import './SellModal.css'
+
+enum StepperValues {
+  SELL_MODAL = 'SELL_MODAL',
+  CONFIRM_INPUT = 'CONFIRM_INPUT',
+  AUTHORIZE = 'AUTHORIZE'
+}
+
+const SellModal = ({
+  wallet,
+  metadata: { nft, order },
+  onClose,
+  getContract,
+  onCreateOrder,
+  isCreatingOrder,
+  authorizations,
+  isAuthorizing,
+  error
+}: Props) => {
+  const { orderService } = VendorFactory.build(nft.vendor)
+
+  const [confirmedInput, setConfirmedInput] = useState<string>('')
+
+  const [stepper, setStepper] = useState(StepperValues.SELL_MODAL)
```

Comment:
If I'm not wrong, this defines the `step` of the `stepper`, what do you think about calling it step?
```suggestion
  const [step, setStep] = useState(StepperValues.SELL_MODAL)
```